cmd/action/internal/build/build.go

```diff
@@ -0,0 +1,19 @@
+package build
+
+import (
+	"runtime/debug"
+)
+
+// Version is dynamically set by the release script.
+var Version = "DEV"
+
+// Date is dynamically set at build time in the release script.
+var Date = "" // YYYY-MM-DD
```

Comment:
```suggestion
var (
	// Version is dynamically set by the release script.
	Version = "DEV"
	// Date is dynamically set at build time in the release script.
	Date = "" // YYYY-MM-DD
)
```
Why the Date is needed?

Reply:
saw it in other CLI's (showing version + date) - have a look in the design under - what others do